core/arch/arm/mm/core_mmu.c

```diff
@@ -860,10 +980,11 @@ static size_t collect_mem_ranges(struct tee_mmap_region *memory_map,
 		add_phys_mem(memory_map, num_elems, &m, &last);
 	}
 
-	if (IS_ENABLED(CFG_SECURE_DATA_PATH))
-		verify_special_mem_areas(memory_map, num_elems,
-					 phys_sdp_mem_begin,
-					 phys_sdp_mem_end, "SDP");
+#ifdef CFG_SECURE_DATA_PATH
```

Comment:
We try to use the `IS_ENABLED()` macro instead of ifdefs where possible.

Reply:
ok, however it will need to define the verify_special_mem_areas function whatever CFG_SECURE_DATA_PATH is set or not.
which introduce unnecessary code if CFG_SECURE_DATA_PATH is not defined.